internal/vds/vds_test.go

```diff
@@ -425,6 +425,61 @@ func TestFence(t *testing.T) {
 	}
 }
 
+func TestFenceBorders(t *testing.T) {
+	testcases := []struct {
+		coordinate_system int
+		coordinates       [][]float32
+		interpolation     string
+		error             string
+	}{
+		{
+			coordinate_system: CoordinateSystemAnnotation,
+			coordinates:       [][]float32{{3, 10}, {6, 11.25}},
+			interpolation:     "nearest",
+			error:             "Coordinate (6.000000,11.250000) is out of boundaries in dimension 0.",
+		},
+		{
+			coordinate_system: CoordinateSystemAnnotation,
+			coordinates:       [][]float32{{5.5, 11.5}, {3, 10}},
+			interpolation:     "linear",
+			error:             "Coordinate (5.500000,11.500000) is out of boundaries in dimension 1.",
```

Comment:
Can we not assert on the numbers in the error message like this? It makes it extremely cumbersome to change stuff